tests/e2e-playwright/pages/admin/courses/courses.ts

```diff
@@ -0,0 +1,30 @@
+const PostType = require('../post-type');
+
+class WizardModal {
+	constructor( locator: Locator ) {
+		this.wizard = locator;
```

Comment:
Declare properties here and in the Dashboard class (from Gabriel's feedback):
<img width="899" alt="Screenshot 2022-12-08 at 19 23 30" src="https://user-images.githubusercontent.com/329356/206633954-abf4ad53-d359-48e6-bbd4-bacc02d5e4eb.png">